100mslive/100ms-flutter
# Sdk/flash capabilities

# Description

Added flash capabilities in android and iOS.
Fixed local peer tile dialog UI bugs.

Closes:

- #1232 
- #1230 

## Additional context

How to test :

- On the local peer tile, click on the three dots.
- If camera is muted then you should only see, two options `change name` and `change role`
- If camera is unmuted then you should see options like `Toggle flash`, `Capture Snapshot`, `Local Image Capture` and `Toggle Camera` along with above options.
- `Toggle Flash` should only work while currently facing camera supports flash and also video is unmuted. In other cases you should get appropriate error toasts.

### Pre-launch Checklist

- [ ] The [Documentation] is updated accordingly, or this PR doesn't require it.


[Documentation]: https://www.100ms.live/docs
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
